packages/web-components/fast-components/src/patterns/start-end.ts

```diff
@@ -18,7 +18,7 @@ export class StartEnd {
 }
 
 export const endTemplate = html<StartEnd>`
-    <span name="end" part="end" ${ref("endContainer")}>
+    <span class="end" name="end" part="end" ${ref("endContainer")}>
```

Comment:
What's the point of the `name` attribute on the span in this case? I'm inclined to think that was supposed to be "class" begin with. Anyone know?